plugins/slick.draggablegrouping.js

```diff
@@ -125,6 +125,20 @@
               return;
             }
             var reorderedIds = $headers.sortable("toArray");
+            // If frozen columns are used, headers has more than one entry and we need the ids from all of them.
+            // though there is only really a left and right header, this will work even if that should change.
+            if($headers.length > 1) {
+              for(var ctr=1,l=$headers.length; ctr < l; ctr+=1) {
```

Comment:
what does `ctr` stand for? Could you provide a better and perhaps longer name to make it more explicit?

Reply:
I just changed ctr and ctr2 to headerI and headerI2

Hopefully the I on the end of headerI and headerI2 are ok given the for loop below the headers.length if uses var i=0 ...
I was trying to avoid redeclaring i as a variable.

For general information: ctr is an abbreviation for counter